service/controller/app/resource/appfinalizer/error.go

```diff
@@ -0,0 +1,12 @@
+package appfinalizer
```

Comment:
```suggestion
package appfinalizermigration
```

Like I mentioned in the issue I think we should have migration in the name.